CalibTracker/SiStripChannelGain/BuildFile.xml

```diff
@@ -0,0 +1,35 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/ParameterSet"/>
+<use name="DQMServices/Core"/>
+<use name="CalibTracker/SiStripCommon"/>
+<use name="CalibFormats/SiStripObjects"/>
+<use name="CalibTracker/Records"/>
+<use name="CondFormats/SiStripObjects"/>
+<use name="CondFormats/RunInfo"/>
+<use name="DataFormats/DetId"/>
+<use name="DataFormats/FEDRawData"/>
+<use name="DataFormats/GeometrySurface"/>
+<use name="DataFormats/SiStripCluster"/>
+<use name="DataFormats/SiStripDetId"/>
+<use name="DataFormats/TrackReco"/>
+<use name="DataFormats/TrackerRecHit2D"/>
+<use name="DataFormats/TrajectorySeed"/>
+<use name="DataFormats/MuonReco"/>
+<use name="RecoTracker/DeDx"/>
+<use name="Geometry/CommonDetUnit"/>
+<use name="Geometry/CommonTopologies"/>
+<use name="Geometry/Records"/>
+<use name="Geometry/TrackerGeometryBuilder"/>
+<use name="Geometry/TrackerNumberingBuilder"/>
+<use name="TrackingTools/PatternTools"/>
+<use name="rootcore"/>
+<use name="root"/>
```

Comment:
Hi @davidlange6 probably it is not, I just moved the includes that were in the pre-existing BuildFile in  `CalibTracker/SiStripChannelGain/plugins/BuildFile.xml`. 
If you wish I can profit of this to clean up.